common/network/UnixSocket.cxx

```diff
@@ -0,0 +1,76 @@
+/* Copyright (C) 2017 Peter Åstrand for Cendio AB.  All Rights Reserved.
+ * 
+ * This is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ * 
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this software; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307,
+ * USA.
+ */
+
+#ifndef WIN32
+
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
+#define closesocket close
```

Comment:
Why? :)

Reply:
Because TcpSocket does it...

Reply:
But it does it conditionally. And it's only for Windows compatibility, which isn't needed here.

Reply:
Correct, but it still makes the destructor of UnixSocket look like TcpSocket, which is nice I think. But in any case, I'll exclude UnixSocket for now.